src/components/SequenceCard/SequenceCard.js

```diff
@@ -55,97 +57,133 @@ export default function SequenceCard({
-            </TimecodeIn>
-            <TimecodeOut>
-              Bauchbinde OUT:
-              <span>
-                {getLowerThirdTimeCodeOut(
-                  index,
-                  allSequenceCards,
-                  timeCodeLowerThirdIn,
-                  timeCodeLowerThirdLength
-                )}
-              </span>
-            </TimecodeOut>
-          </LowerThirdContainer>
-        </InfoIfTag>
-      ) : (
-        <Timecode>{getFormatedTimecode(lengthTimeCode)}</Timecode>
+            <Player>{playerName}</Player>
+          </TagAndPlayer>
+          <Line />
+          <LowerThirds>
+            <InfoAndTimecode
+              info="Bauchbinde in"
+              timecode={getLowerThirdTimeCodeIn(
+                index,
+                allSequenceCards,
+                timeCodeLowerThirdIn
+              )}
+            />
+            <InfoAndTimecode
+              info="bauchbinde out"
+              timecode={getLowerThirdTimeCodeOut(
+                index,
+                allSequenceCards,
+                timeCodeLowerThirdIn,
+                timeCodeLowerThirdLength
+              )}
+            />
+          </LowerThirds>
+        </AdditionalContent>
       )}
+
+      <CardFooter>
+        <EditButton
+          onClick={() => {
+            handleOnEdit()
+            history.push('/create')
+          }}
+        />
+      </CardFooter>
     </Card>
   )
 }
 
 const Card = styled.section`
-  border-radius: 10px;
+  border-radius: 30px;
   border: 1px solid grey;
+  color: white;
+  ${(props) => !props.isActive && 'opacity: 0.3;'}
   margin: 15px 20px;
-  padding: 10px 10px;
-  ${(props) => !props.isActive && 'background-color : #E7C5CA; color: grey;'}
+  padding: 20px 30px;
+  background: transparent linear-gradient(180deg, #001496 0%, #0032c8 100%) 0%
+    0% no-repeat padding-box;
 `
 
+const MainContent = styled.section`
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+`
 const Description = styled.p`
+  font-family: 'BaiJamjuree';
+  font-size: 1em;
   word-break: break-word;
+  color: white;
 `
 const Header = styled.header`
   display: flex;
   justify-content: space-between;
+  align-items: center;
 `
-const Timecode = styled.p`
-  font-weight: 800;
+
+const Minute = styled.h2`
+  font-family: 'BaiJamjuree';
+  font-size: 2em;
   margin: 0;
-  text-align: right;
 `
 
-const InfoIfTag = styled.div`
+const AdditionalContent = styled.section`
   display: flex;
   flex-direction: column;
+  padding: 0;
+  margin-top: 25px;
 `
-const TagAndLength = styled.div`
-  align-items: center;
+const TagAndPlayer = styled.div`
   display: flex;
-  justify-content: space-between;
+  align-items: center;
 `
 
-const Tag = styled.div`
-  background-color: var(--tag-border-grey);
-  border-radius: 5px;
-  color: #8e8e8e;
-  height: 30px;
-  padding: 5px 6px;
+const Player = styled.p`
+  font-family: 'BaiJamjuree';
+  font-size: 0.8em;
+  margin: 0;
+  padding: 0;
+  color: #00dc8c;
```

Comment:
You could add this to your GlobalStyles colors as well. Even if this is a singular use, giving it a name makes the code more readable than seeing just a hex code.